tests/layering/node_cmd_test.go

```diff
@@ -0,0 +1,436 @@
+package e2e_test
```

Comment:
I wanted to mention that I may want to eventually port this library to the MCO's testing library. With this present, we could reduce our need to shell out to `oc` to execute arbitrary commands on a node.

Reply:
good idea, along those lines, can you add an owners file to this tests/layering?